integration

```diff
@@ -603,6 +603,7 @@
   "jcgoette/weight_gurus_homeassistant",
   "jcwillox/hass-auto-backup",
   "jdejaegh/irm-kmi-ha",
+  "jdolieslager/homeassistant-dynamic-dns",
```

Comment:
Your documentation goes nowhere (404)
https://github.com/jdolieslager/homeassistant-dynamic-dns/blob/v0.0.1/custom_components/dynamic_dns/manifest.json#L7C21-L7C58
And also points to some other users github account.